deploy/helm/kubecf/templates/_stacks.tpl

```diff
@@ -0,0 +1,138 @@
+  {{- end }}
+  {{- if not (index $stack.releases $cc_stack.name) }}
+    {{- $_ := set $stack.releases $cc_stack.name dict }}
+  {{- end }}
+
+  {{- /* *** Create "$stack.install_buildpacks" and "$stack.releases" from the "api" buildpack jobs *** */}}
+  {{- $_ := set $stack "install_buildpacks" list }}
+
+  {{- $_ := include "_config.lookupManifest" (list $ "instance_groups/name=api/jobs") }}
+  {{- range $job := $.kubecf.retval }}
+    {{- if hasSuffix "-buildpack" $job.release }}
+      {{- $buildpack_shortname := trimSuffix "-buildpack" $job.release }}
+      {{- $_ := set $stack "install_buildpacks" (append $stack.install_buildpacks $buildpack_shortname) }}
+      {{- /* Make sure an entry for the release exists; versions will be filled in by _releases.update */}}
+      {{- if not (index $stack.releases $job.release) }}
+        {{- $_ := set $stack.releases $job.release dict }}
+      {{- end }}
+    {{- end }}
+  {{- end }}
+
+  {{- /* +----------------------------------------------------------------------------------------------+ */}}
+  {{- /* | Setup the additional stacks in .config.stacks and merge their releases into .config.releases | */}}
+  {{- /* +----------------------------------------------------------------------------------------------+ */}}
+
+  {{- $_ := include "_config.lookup" (list $ "stacks") }}
+  {{- range $stack_name, $stack := $.kubecf.retval }}
+    {{- $_ := set $stack "enabled" (has $stack_name $.Values.install_stacks) }}
+
+    {{- /* *** Update all releases with defaults from stack.releases.$defaults *** */}}
+    {{- $_ := include "_releases.applyDefaults" $stack.releases }}
+
+    {{- /* *** Combine install_buildpacks with prepend and append lists *** */}}
+    {{- $install_buildpacks := list }}
+    {{- if $stack.install_buildpacks_prepend }}
+      {{- $install_buildpacks = $stack.install_buildpacks_prepend }}
+    {{- end }}
+    {{- if $stack.install_buildpacks }}
+      {{- $install_buildpacks = concat $install_buildpacks $stack.install_buildpacks }}
+    {{- end }}
+    {{- if $stack.install_buildpacks_append }}
+      {{- $install_buildpacks = concat $install_buildpacks $stack.install_buildpacks_append }}
+    {{- end }}
+    {{- $_ := set $stack "install_buildpacks" $install_buildpacks }}
+
+    {{- /* create $stack.buildpacks mapping from buildpack shortname to release name */}}
+    {{- $_ := set $stack "buildpacks" dict }}
+
+    {{- $release_prefix := printf "%s-" (default "" $stack.release_prefix) }}
+    {{- $release_suffix := printf "-%s" (default "buildpack" $stack.release_suffix) }}
+    {{- range $release_name, $release := $stack.releases }}
+      {{- $_ := set $release "condition" $stack.enabled }}
+
+      {{- /* *** Set buildpack shortname from release name (unless already set, or rootfs) *** */}}
+      {{- if and (ne $release_name $stack_name) (not $release.buildpack) }}
+        {{- $shortname := ($release_name | trimSuffix "-buildpack") }}
```

Comment:
Hmm, why do we need the extra `trimSuffix` here again? The next line would do it (in the default case) already, if I'm reading this right.

Reply:
Fixed in fc383dd